src/protocol/modulus_conversion/double_random.rs

```diff
@@ -106,7 +99,7 @@ impl DoubleRandom {
         record_id: RecordId,
         a: Replicated<F>,
         b: Replicated<F>,
-    ) -> Result<Replicated<F>, Error> {
+    ) -> Result<Replicated<F>, BoxError> {
```

Comment:
is something not working with `Error`?

Reply:
Discussed offline. Seems like we could slowly migrate everything to Error. Maybe not this diff tho.